topics/ecology/tutorials/Ecoregionalization_tutorial/tutorial.md

```diff
@@ -439,6 +439,12 @@ Here you will load an existing jupyter notebook ("pivot_wider_jupytool_notebook.
 >
 > ![Interactive JupyterLab Notebook output example](./Images/pivot_file_example.png "Interactive JupyterLab Notebook output example")
 >
+> >  <warning-title>Galaxy can't display properly file with too many columns</warning-title>
```

Comment:
@yvanlebras is there already an issue filed for this on the Galaxy repo? Sounds like a bug

Reply:
THANK YOU Saskia! Yes, I created one! https://github.com/usegalaxy-eu/galaxy/issues/244 It seems wonder @bgruening updated the image and then, jupytool launched as a classical tool was not working properly with the last version of kernels.